test/test_helper.exs

```diff
@@ -1,2 +1,2 @@
 ExUnit.start()
-ExUnit.configure(exclude: [:pending, :external], trace: true)
+ExUnit.configure(exclude: [:pending, :external])
```

Comment:
It's super annoying when the tests run with --trace, I can never find the failure then

Reply:
Ah, yes, that is a quality of life improvement :)